packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -226,449 +226,598 @@ import 'runtime.dart';
 /// argument and `curve` argument. It sets the default animation duration and
 /// curve for widgets in the library that use the animated variants. If absent,
 /// a default of 200ms and [Curves.fastOutSlowIn] is used.
-LocalWidgetLibrary createCoreWidgets() => LocalWidgetLibrary(_coreWidgetsDefinitions);
+LocalWidgetLibrary createCoreWidgets() =>
+    LocalWidgetLibrary(_coreWidgetsDefinitions);
 
 // In these widgets we make an effort to expose every single argument available.
-Map<String, LocalWidgetBuilder> get _coreWidgetsDefinitions => <String, LocalWidgetBuilder>{
```

Comment:
can you keep the original formatting? it's much, much easier to review changes with the old formatting than with what dart format does.

Reply:
Reverted. Should I format it in another pr or it's ok to just keep it what it is?